sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_sender.py

```diff
@@ -169,7 +169,7 @@ def _from_connection_string(
         :rtype: ~azure.servicebus.ServiceBusSender
 
         :raises ~azure.servicebus.ServiceBusAuthenticationError: Indicates an issue in token/identity validity.
-        :raises ~azure.servicebus.ServiceBusAuthorizationError: Indicates an access/rights related failure.
+        :raises ~azure.servicebus.UnauthorizedAccessError: Indicates an access/rights related failure.
```

Comment:
@annatisch from OOB convo with Adam, this was motivated by normalization with dotnet, but it doesn't sit well with me, especially in line with our broader leaning towards prefixing especially for ambiguous errors. (To be precise, I'd revert to ServiceBusAuthorization, or, if the alignment was desirable, meet in the middle with (Oh goodness.) ServiceBusUnauthorizedAcceessError)  How do you feel about this one?

Reply:
as per oob discussion:
UnauthorizedAccessError back to ServiceBusAuthorizationError